app/src/main/java/com/fstyle/structure_android/data/source/local/UserLocalDataSource.java

```diff
@@ -0,0 +1,249 @@
+package com.fstyle.structure_android.data.source.local;
+
+import android.content.ContentValues;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.support.annotation.NonNull;
+import com.fstyle.structure_android.data.model.User;
+import com.fstyle.structure_android.data.source.UserDataSource;
+import com.fstyle.structure_android.data.source.local.config.sqlite.UserDbHelper;
+import io.reactivex.Completable;
+import io.reactivex.CompletableEmitter;
+import io.reactivex.CompletableOnSubscribe;
+import io.reactivex.Maybe;
+import io.reactivex.MaybeEmitter;
+import io.reactivex.MaybeOnSubscribe;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by daolq on 1/10/18.
+ */
+
+public class UserLocalDataSource implements UserDataSource.LocalDataSource {
+
+    private static final String SELECT_ALL_USER_QUERY =
+            String.format("SELECT * FROM %s", UserDbHelper.UserEntry.TABLE_NAME);
+
+    private static UserLocalDataSource INSTANCE;
```

Comment:
WARNING: [SuspiciousConstantFieldName] The field name indicates a constant but its modifiers do not
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/naming.html#SuspiciousConstantFieldName